pkg/graph/resolvers/system_intake_grb_review_form.go

```diff
@@ -92,6 +92,7 @@ func UpdateSystemIntakeGRBReviewFormInputTimeframeAsync(
 	}
 
 	intake.GrbReviewAsyncEndDate = &input.GrbReviewAsyncEndDate
+	intake.GrbReviewAsyncManualEndDate = nil
```

Comment:
@OddTomBrooks, were you thinking this would be used for the restart voting endpoint? I'm not confident if this should always be cleared when this endpoint is hit. 

I wonder if we might need a separate endpoint for restarting voting, or a parameter that specifies if we are restarting voting here. (I think the main differences will be clearing the date and sending an email.) What do you think?

Reply:
This is in accordance with specification by the AC on the ticket. @ClayBenson94 Do you recall the context around that discussion?

Reply:
Fair enough, for the sake of the current use, I think we can merge as is. We might want to revisit this later, as I think that there are a separate emails sent depending on the use case.